src/contexts/Guilds/filters.tsx

```diff
@@ -110,6 +110,7 @@ export const FilterProvider = ({
   const withFilters =
     (Component: JSX.Element) =>
     (proposal: Proposal, proposalState: ProposalState, options: Option[]) => {
+      if (!options || options.includes(undefined)) return null;

```

Comment:
I don't think we should do this. If at some point options is undefined (options still loading) we want to show proposals. If we do this nothing will be returned. (my opinion)
I think you may found an issue but is regarding the next line:
`const actions = options?.map(option => option?.decodedActions).flat();`

Reply:
Alright! Thanks for noticing this!